fmriprep/run_workflow.py

```diff
@@ -116,18 +116,19 @@ def main():
         os.makedirs(settings['output_dir'])
 
     # Logging and work directory
-    if opts.work_dir:
-        settings['work_dir'] = op.abspath(opts.work_dir)
-
-        log_dir = op.join(settings['work_dir'], 'log')
-        if not op.exists(log_dir):
-            os.makedirs(log_dir)
-
-        # Set nipype config
-        ncfg.update_config({
-            'logging': {'log_directory': log_dir, 'log_to_file': True},
-            'execution': {'crashdump_dir': log_dir}
-        })
+    settings['work_dir'] = (op.abspath(opts.work_dir)
```

Comment:
This is too verbose, IMHO. For this very same behavior, it is enough to:
1. Define a ```default=os.getcwd()``` in the argument (line 95)
2. Do ```settings['work_dir'] = op.abspath(opts.work_dir)